web-ui/src/pages/ViewFeedbackPage.jsx

```diff
@@ -134,20 +136,14 @@ const ViewFeedbackPage = () => {
       const contains = (toFind) => feedbackRequests.findIndex(request => request.id === toFind.id) !== -1;
 
       if(teamMembers && teamMembers.length > 0) {
-        await Promise.all(teamMembers.map((member) => {
-          return new Promise(async (resolve) => {
-            const memberRequests = await getFeedbackRequestsById(member.id);
-            memberRequests.forEach((request) => {
-              if (!contains(request)) {
-                feedbackRequests.push(request);
-              }
-            });
-            resolve();
-          });
-        }));
+        const memberRequests = await getFeedbackRequestsForTeamMembers(teamMembers);
+        memberRequests.forEach((request) => {
+          if (!contains(request)) {
+            feedbackRequests.push(request);
```

Comment:
Maybe `feedbackRequests` should be a `Set` so it will automatically exclude duplicates.

Reply:
This is probably exposing how little I write code these days, but is there a way to provide Set with a custom comparator (see line 134/136)? Or is there some reason that wouldn't be necessary that I'm missing?

Reply:
I don't think there is a way to do that. I believe it compares primitive values by their values and objects by their address in memory. So I think a Set could contain two objects with identical properties as long as they are distinct objects in memory.